src/libpriv/rpmostree-core.cxx

```diff
@@ -4435,6 +4435,17 @@ rpmostree_context_assemble (RpmOstreeContext *self, GCancellable *cancellable, G
 
   CXX_TRY_VAR (etc_guard, rpmostreecxx::prepare_tempetc_guard (tmprootfs_dfd), error);
 
+  /* if the %__systemd_sysusers macro is defined, that _very likely_
+   * means that we're using an rpm with built-in sysusers support:
+   * https://fedoraproject.org/wiki/Changes/RPMSuportForSystemdSysusers Ideally,
```

Comment:
Minor/optional: I know I wrote this, but just noticed it's missing a period here. :)